s3-source-connector/src/integration-test/java/io/aiven/kafka/connect/s3/source/AwsIntegrationTest.java

```diff
@@ -121,10 +139,10 @@ private Map<String, String> getConfig(final String topics, final int maxTasks) {
      */
     @Test
     void sourceRecordIteratorBytesTest(final TestInfo testInfo) {
-        final var topicName = IntegrationBase.topicName(testInfo);
+        final var topic = IntegrationBase.getTopic(testInfo);
         final int maxTasks = 1;
         final int taskId = 0;
-        final Map<String, String> configData = getConfig(topicName, maxTasks);
+        final Map<String, String> configData = getConfig(topic, maxTasks);
 
         configData.put(INPUT_FORMAT_KEY, InputFormat.BYTES.getValue());
         configData.put(FILE_NAME_TEMPLATE_CONFIG, "{{topic}}-{{partition}}-{{start_offset}}");
```

Comment:
Not related here in this file, but DEFAULT_FILE_PATH_PREFIX_TEMPLATE , getFilePathPrefixTemplateConfig() has no usages. Can we delete that ?